tests/ICU4N.Tests/Dev/Test/Util/ICUResourceBundleTest.cs

```diff
@@ -13,7 +13,12 @@ namespace ICU4N.Dev.Test.Util
 {
     public sealed class ICUResourceBundleTest : TestFmwk
     {
-        private static readonly Assembly testLoader = typeof(ICUResourceBundleTest).GetTypeInfo().Assembly; //ICUResourceBundleTest.class.getClassLoader();
+        private static readonly Assembly testLoader =
+#if FEATURE_TYPEEXTENSIONS_GETTYPEINFO
+        typeof(ICUResourceBundleTest).GetTypeInfo().Assembly; //ICUResourceBundleTest.class.getClassLoader();
```

Comment:
Please indent this 4 spaces so it is clear it is a line continuation, not a new line.

Reply:
Okay, i will fix this in the next commit.